src/v/model/fundamental.h

```diff
@@ -238,6 +238,26 @@ class materialized_ntp {
     const model::ntp& _source_or_input;
 };
 
+/**
+ * Enum describing Kafka's control record types. The Control record key schema
+ * is defined as a tuple of [version: int16_t, type: control_record_type].
+ * Currently the only supported version of control record is equal to '0'.
```

Comment:
what does `Currently the only supported version of control record is equal to '0'.` mean?

Reply:
I've updated the comment to be more clear. The point is that the version currently defined by Kafka is `0`